emily/handler/src/api/handlers/deposit.rs

```diff
@@ -45,11 +46,13 @@ use crate::database::entries::deposit::{
         (status = 500, description = "Internal server error", body = ErrorResponse)
     )
 )]
+#[instrument(skip(context))]
 pub async fn get_deposit(
     context: EmilyContext,
     bitcoin_txid: String,
     bitcoin_tx_output_index: u32,
 ) -> impl warp::reply::Reply {
+    debug!("In get deposit");
```

Comment:
They're intentional. I think it makes sense to leave these in because they're at debug level. If we want to stop printing this much then we would want to set the log level to be info over debug.